myapp/spec/models/task_spec.rb

```diff
@@ -27,4 +27,48 @@
       ).with_prefix
     }
   end
+
+  describe '.sort_by_keyword' do
+    let!(:first_task) { create(:task, name: 'ううう') }
+    let!(:second_task) { create(:task, name: 'あああ') }
+    let!(:third_task) { create(:task, name: 'いいい') }
+
+    subject { Task.sort_by_keyword(sort) }
+
+    context 'when sort type is created_at_asc' do
+      let(:sort) { 'created_at_asc' }
+
+      it 'sort by specified sort type' do
+        is_expected.to eq [first_task, second_task, third_task]
+      end
+    end
+
+    context 'when sort type is created_at_desc' do
+      let(:sort) { 'created_at_desc' }
+
+      it 'sort by specified sort type' do
+        is_expected.to eq [third_task, second_task, first_task]
+      end
+    end
+  end
+
+  describe '.check_approved_sort_params' do
+    subject { Task.check_approved_sort_params(sort) }
+
+    context 'sort type: name_asc' do
```

Comment:
細かいですがどちらかというと「`Task::SORT_TYPE`に存在しないパラメータの場合」みたいな書き方の方がテストケースの意味合い的には正しそうですー

```suggestion
    context "When argument 'sort' does NOT exist in Task::SORT_TYPE keys" do
```

Reply:
これとは別に`created_at_asc`のケースを作るか、66行目のテストケースを`"When argument 'sort' exists in Task::SORT_TYPE keys"`等で書いた方が良いと思いますー

命令網羅的には後者でも問題無さそうですが、分岐網羅的には前者も行ったほうが良さそうですね

Reply:
> 細かいですがどちらかというと「Task::SORT_TYPEに存在しないパラメータの場合」みたいな書き方の方がテストケースの意味合い的には正しそうですー
- 「Task::SORT_TYPEに存在しないパラメータの場合」で書かせて頂きました！

> これとは別にcreated_at_ascのケースを作るか、66行目のテストケースを"When argument 'sort' exists in Task::SORT_TYPE keys"等で書いた方が良いと思いますー
- created_at_ascのケースを追記する形を取らせて頂きました！